pypesto/optimize/util.py

```diff
@@ -188,3 +189,36 @@ def check_finite_bounds(lb, ub):
             "Selected optimizer cannot work with unconstrained "
             "optimization problems."
         )
+
+
+def laplace_approximation_log_evidence(
+    problem: Problem, x: np.ndarray
+) -> float:
+    """
+    Compute the log evidence using the Laplace approximation.
+
+    Expects the problem to have a Hessian method implemented and the objective to be negative log posterior.
```

Comment:
If we expect a negative log posterior (which is usually found in sampling not in optimization, since there we use mostly likelihoods), why have it in optimize and not in sampling?

Reply:
If you specify priors and then we also use the posterior in optimisation. This method does not require sampling at all. Moreover, you would not use this function with a likelihood alone without specifying priors since the model evidence makes only sense in a Bayesian framework.

Reply:
i still do not see the reason to package it into optimize, since it is really not a utility function of optimize. Quite the opposite, as its usage is in sampling/util.py which for me would make it much clearer in sample

Reply:
this is a standalone function. It is used in sampling/util.py as a method to get starting values, but besides that you can use it to compute marginal likelihoods without sampling

Reply:
will more everything in separate file to make it more organised